bin/gen_make_targets

```diff
@@ -3,7 +3,12 @@
 set -Eeufo pipefail
 
 hash="$(sha256sum flavours.yaml | cut -c -64)"
-check="$(head -n 1 make_targets.cache || true)"
+
+if [ -f "make_targets.cache" ]; then
+	check="$(head -n 1 make_targets.cache)"
+else
+	check=true
```

Comment:
The value `true` can be confusing  her. Otherwise looks good to me

Reply:
Hey @Vincinator,
thanks for your feedback. How'd u like to name it? I think this represents that a "check" should be performed and will be fine in L13 by `if [ "$hash" != "$check" ]; then`.

Reply:
Actually, you are right. If the variable name is `check` then the value `true` actually does make sense. 

I would go for a radical approach for this unimportant detail and rename the variable.
e.g.:
`cached_hash=none`  would be clearer for me.